system-x/services/mail/src/main/java/software/tnb/mail/validation/MailValidation.java

```diff
@@ -0,0 +1,57 @@
+package software.tnb.mail.validation;
+
+import software.tnb.common.utils.HTTPUtils;
+import software.tnb.mail.account.MailAccount;
+
+import org.apache.commons.mail.DefaultAuthenticator;
+import org.apache.commons.mail.Email;
+import org.apache.commons.mail.EmailException;
+import org.apache.commons.mail.SimpleEmail;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MailValidation {
+    private static final Logger LOG = LoggerFactory.getLogger(MailValidation.class);
+
+    private final MailAccount account;
+    private final String host;
+    private final int smtpPort;
+    private final int httpPort;
+
+    private final String subject;
+
+    public MailValidation(MailAccount account, String host, int smtpPort, int httpPort) {
+        this.account = account;
+        this.host = host;
+        this.smtpPort = smtpPort;
+        this.httpPort = httpPort;
+        subject = "TestMail from TNB " + System.currentTimeMillis();
+    }
+
+    public void sendEmail(String message) {
```

Comment:
maybe also a method with `String subject, String message` could be better, instead of hardcoding subject in constructor?